docs/performance.md

```diff
@@ -0,0 +1,254 @@
+# Tessera Storage Performance
+
+This document describes the performance of each Trillian Tessera storage implementation under different conditions.
+
+## MySQL
+
+### GCP Free Tier VM Instance
+
+**e2-micro**
```

Comment:
I think we should put caveats in here since the `e2-micro/small/medium` machines are all  "shared core", and don't have local storage for MySQL etc. - I suspect these numbers will vary wildly depending on local "weather".

It's great that Tessera _can_ run in the free tier on these machines, and it's worth calling that out, but we should also be clear about what folks would be getting if they do so.

Reply:
Added a note about the potential performance impact using vCPUs shared core.

Reply:
Mentiond Tessera can run on GCP free tier.